### PR TITLE
MPP-3153 - add stripe.js

### PR DESCRIPTION
**Pull request by lloan**
This PR fixes MPP-3153
 

# New feature description

To best leverage Stripe’s advanced fraud functionality, include this script on every page, not just the checkout page. This [allows Stripe to detect suspicious behavior](https://stripe.com/docs/disputes/prevention/advanced-fraud-detection) that may be indicative of fraud as customers browse your website.

# Screenshot (if applicable)

image

# How to test

Load the branch and inspect the page (any page). You should see the following: 

image

# Checklist (Definition of Done)
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [ ] All acceptance criteria are met.
- [ ] Jira ticket has been updated (if needed) to match changes made during the development process.
- [ ] I've added or updated relevant docs in the docs/ directory
- [ ] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.
- [ ] All UI revisions follow the [coding standards](https://github.com/mozilla/fx-private-relay/blob/main/docs/coding-standards.md), and use Protocol tokens where applicable (see `/frontend/src/styles/tokens.scss`).
- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [ ] l10n changes have been submitted to the l10n repository, if any.

### CODE REVIEW

**Review by Vinnl — APPROVED** [review]
I assume legal agrees with this? This does give Stripe full keys to the kingdom, giving them the technical ability to quite extensively track our users and/or to introduce vulnerabilities - so that should be covered by legal assurances.

File: privaterelay/settings.py [hunk 1]
```diff
@@ -105,10 +105,11 @@
     "https://location.services.mozilla.com",
     BASKET_ORIGIN,
 )
-CSP_DEFAULT_SRC = ("'self'",)
+CSP_DEFAULT_SRC = ("'self'", "https://js.stripe.com/")
```
Comment (jwhitlock) [inline privaterelay/settings.py:108]: I don't think `CSP_DEFAULT_SRC` needs to be changed, we have more specific directives that will override it.

**Review by jwhitlock — CHANGES_REQUESTED** [review]
@lloan Nice work remembering the CSP changes!

I _think_ we need `api_stripe.com` in `CSP_CONNECT_SRC` as well, but your testing may show differently.

I'd be OK either way on `https://maps.googleapis.com` and `CSP_FRAME_SRC` - include them now or wait until we see CSP errors in stage

**Review by jwhitlock — APPROVED** [review]
Thanks @lloan!

**Pull request merged**